client/components/language-picker/test/index.jsx

```diff
@@ -27,16 +29,6 @@ const languages = [
 ];
 
 describe( 'LanguagePicker', () => {
-	let LanguagePicker;
-
-	useMockery( mockery => {
-		mockery.registerMock( 'lib/analytics', {} );
-	} );
-
-	before( () => {
-		LanguagePicker = require( '../' );
```

Comment:
This was one of the most annoying side effect of using mockery. Another one was accidental timeouts ...